packages/features/insights/components/FailedBookingsByField.tsx

```diff
@@ -0,0 +1,93 @@
+import { BarList } from "@tremor/react";
+import { useState } from "react";
+
+import { classNames } from "@calcom/lib";
+import { trpc } from "@calcom/trpc";
+import { ToggleGroup } from "@calcom/ui";
+
+import { useFilterContext } from "../context/provider";
+
+interface FormCardProps {
+  formName: string;
+  fields: Record<string, { optionId: string; count: number; optionLabel: string }[]>;
+}
+
+function FormCard({ formName, fields }: FormCardProps) {
+  const fieldNames = Object.keys(fields);
+  const [selectedField, setSelectedField] = useState(fieldNames[0]);
+
+  const toggleOptions = fieldNames.map((fieldName) => ({
+    value: fieldName,
+    label: fieldName.charAt(0).toUpperCase() + fieldName.slice(1),
+  }));
+
+  const selectedFieldData = fields[selectedField]?.map((option) => ({
+    name: option.optionLabel,
+    value: option.count,
+  }));
+
+  return (
+    <div className="border-subtle h-[400px] w-full rounded-md border">
+      <div className="flex flex-col">
+        <div className="bg-subtle border-subtle flex h-12 items-center border-b">
+          <h3 className="text-default px-2 text-left align-middle font-medium">{formName}</h3>
+        </div>
+        <div className="p-4">
+          <ToggleGroup
+            options={toggleOptions}
+            value={selectedField}
+            className="w-fit"
+            onValueChange={(value) => value && setSelectedField(value)}
+          />
+          {selectedFieldData && (
+            <div className="scrollbar-thin mt-4 h-full overflow-y-auto">
+              <BarList
+                data={selectedFieldData}
+                valueFormatter={(value: number) => value.toString()}
+                className="mt-2"
+              />
+            </div>
+          )}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export function FailedBookingsByField() {
+  const { filter } = useFilterContext();
+  const { selectedTeamId, isAll, initialConfig, selectedRoutingFormId } = filter;
+  const initialConfigIsReady = !!(initialConfig?.teamId || initialConfig?.userId || initialConfig?.isAll);
+
+  const { data } = trpc.viewer.insights.failedBookingsByField.useQuery(
+    {
+      teamId: selectedTeamId ?? undefined,
+      isAll: !!isAll,
+      routingFormId: selectedRoutingFormId ?? undefined,
+    },
+    {
+      enabled: initialConfigIsReady,
+    }
+  );
+
+  if (!data) return null;
+
+  return (
+    <div className="w-full text-sm">
+      <div className="flex h-12 items-center">
+        <h2 className="text-emphasis text-md font-semibold">Failed Bookings By Field</h2>
```

Comment:
NIT: Missing translation

Reply:
I see this comment is resolved, but don't see the actual `t()` function here. Is the change not pushed yet?